### ISSUE TITLE
Multiple Upload Freeze

### ISSUE THREAD

**Issue by danielRL24** [pqina/filepond#437]
Hello,

I am trying to load several documents at the same time with Filepond.

On my Dev server, I don't have any problem, the documents load completely one after the other.

On the Production server, when I try to upload several documents, the upload freeze.
Then after a while (1-2min), the request is canceled and I get the error "_POST net::ERR_HTTP2_PROTOCOL_ERROR_" in the console (with Chrome). With Firefox, no error in console and the request is just canceled.
As soon as I remove an item (any item) from the list, the loading resumes.

The error callback doesn't give me any more information (the error parameter is empty and the status is undefined).

Has anyone ever had the same problem before? Does anyone have any idea where to look?

I use the latest version of Filepond with the jQuery adapter. I use the standard processing method (like the example in the documentation)

Thank you in advance.

**Comment by rikschennink** [comment]
I suggest to take a look at the network tab of your dev tools to look up the upload request and see if it gives any further details. This is most likely a server issue.

**Comment by danielRL24** [comment]
Hi,

After analyzing the exchanges in the "Network" tab, I noticed that my Development server uses the HTTP/1.1 protocol and my Production server uses the HTTP/2 protocol.

I performed tests on two other servers and I arrive at the same result. File uploading works correctly on the one using the HTTP/1.1 protocol and freezes on the one using the HTTP/2 protocol.

I also tested by disabling HTTP/2 on Firefox and called the problematic server and it worked. This makes me say that the problem is indeed related to the HTTP/2 protocol.

Have you encountered this problem before ? 
Is there any configuration that needs to be done specifically for Filepond to work with HTTP/2 ?
(knowing that I can't disable HTTP/2 protocol on my production server)

Thank you in advance.
